src/main/webapp/index.jsp

```diff
@@ -33,7 +33,10 @@
               <%
               if (userLoggedIn && userService.isUserAdmin()) { 
               %>
-                  <a href="admin_dashboard.html">
+                  <a href="user_profile.jsp" class="user-icon">
```

Comment:
nit: I think calling the class as `dashboard-icon` can be a better idea because all the icons lead to a dashboard, either a dashboard for users or a dashboard for admins.

Reply:
I'll make this change, thanks!